lib/resourceful_endpoint.js

```diff
@@ -212,7 +212,67 @@ class ResourcefulEndpoint {
    * @returns {Promise} - A {@link ResourceCollection}
    */
   browse(criteria, options) {
-    return this.transport.get(this.endPointUrl(null, criteria), options).then(json => new ResourceCollection(json, criteria, this));
+    return this.transport
+      .get(this.endPointUrl(null, this.getRelatedThroughFields(criteria)), options)
+      .then(json => new ResourceCollection(json, criteria, this));
+  }
+
+  /**
+   * Private
+   * Look through the relationships of a criteria, and include fields from through
+   * relationships - this makes it so that the end user shouldn't know or care that
+   * through relationships exist.
+   *
+   * @param {(string|Query)} criteria - A query string to append to the request
+   *
+   * @returns {(Query)} criteria - A qquery with additional includes if needed
+
+   */
+  getRelatedThroughFields(criteria) {
+    if (!criteria || !criteria.query) return criteria;
+    const query = criteria.query || '';
+    const params = query.split('&');
```

Comment:
Could this use the `qs` lib, rather than splitting strings?

Reply:
This would make sense